config/core/test-configs.yaml

```diff
@@ -1616,7 +1624,6 @@ test_configs:
       - baseline
       - ltp-crypto
       - ltp-ipc
-      - ltp-mm
```

Comment:
Please move this to a separate PR, it's true we should stop running `ltp-mm` on `bcm2836-rpi-2-b` because it's taking too long to complete.  But it's not related to this commit about timers.